$OpenBSD: patch-lib_Xm_PanedW_c,v 1.1 2010/02/17 18:15:09 jasper Exp $

Causes problems, this patch dropped from HEAD OpenMotif. See bug 1476.

--- lib/Xm/PanedW.c.orig	Wed Feb 17 17:34:45 2010
+++ lib/Xm/PanedW.c	Wed Feb 17 17:34:53 2010
@@ -60,7 +60,7 @@ static char rcsid[] = "$TOG: PanedW.c /main/24 1999/07
 #include "MessagesI.h"
 #include "RepTypeI.h"
 
-#define FIX_1476
+// #define FIX_1476
 
 typedef enum {FirstPane='U', LastPane='L'} Direction;
 
